infra/modules/ecs_service/main.tf

```diff
@@ -1,25 +1,25 @@
 module "log_router_container" {
-  source    = "git::github.com/wellcomecollection/terraform-aws-ecs-service.git//modules/firelens?ref=v3.13.1"
+  source    = "git::https://github.com/wellcomecollection/terraform-aws-ecs-service.git//modules/firelens?ref=v3.13.1"
```

Comment:
Get modules via HTTPs to avoid authentication problem with SSH in CI, these repos are public in any case.